styles/addMember.css

```diff
@@ -0,0 +1,52 @@
+form {
+    margin-left: 145px;
+    margin-top: 48px;
+  }
+  
+input[type=text], input[type=date] {
+    width: 576px;
+    height:43px;
+    padding: 12px 16px;
+    align-items: center;
+    margin: 8px 0;
+    border: 2px solid var(--clr-white);
+    border-radius: 7px;
+    gap: 10px;
+    box-sizing: border-box;
+  }
+
+ button  {
+     color: var(--clr-white);
+    padding: 12px 24px;
+    border: none;
+    border-radius: 7px;
+    font-weight: 600;
+    margin-left: 428px;
```

Comment:
You're selecting ***every*** single button that will be in this project. While most of the styles here are perfect, the margins won't work here, as they'll apply to ***every*** single button in the project, which is not what we want. You should delete these and read on for more steps